editor/components/dialogs/share-dialog.tsx

```
@@ -0,0 +1,291 @@
+    <Drawer open={open} onOpenChange={onOpenChange} data-testid={testId}>
+      <DrawerTitle className="sr-only">Share</DrawerTitle>
+      <DrawerContent>
+        <div className="mx-auto w-full">
+          {description ? (
+            <div className="p-4">{description}</div>
+          ) : (
+            descriptionHtml && (
+              <article
+                className="p-4 prose prose-sm dark:prose-invert"
+                dangerouslySetInnerHTML={{ __html: descriptionHtml }}
+              />
```

Biome (2.3.13)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)